test/datasets_utils.py

```diff
@@ -734,7 +734,7 @@ def size(idx: int) -> Tuple[int, int, int]:
             return (num_channels, height, width)
 
     root = pathlib.Path(root) / name
-    os.makedirs(root)
+    os.makedirs(root, exist_ok=True)
```

Comment:
I think this is fine. Could you add the same to `create_video_folder()` so they don't diverge?